tests/e2e/secretapp/secretapp_test.go

```diff
@@ -96,49 +105,55 @@ func generateTestCases() []testCase {
 		nil,
 	}
 
-	testCase1Key := "daprsecret"
-	testCase2Key := "daprsecret2"
-	testCase1Value := "admin"
-
 	return []testCase{
 		{
 			// No comma since this will become the name of the test without spaces.
-			"Test get with empty request response for single app and single hop",
-			[]testStep{
-				{
-					"get",
-					emptyRequest,
-					emptyResponse,
-					false,
-					200,
-				},
-			},
+			"empty request",
+			emptyRequest,
+			emptyResponse,
+			false,
+			200,
+			"", // no error
 		},
 		{
-			// No comma since this will become the name of the test without spaces.
-			"Test get a single item for single app and single hop",
-			[]testStep{
-				{
-					"get",
-					newRequest(utils.SimpleKeyValue{testCase1Key, testCase1Value}),
-					newResponse(utils.SimpleKeyValue{testCase1Key, testCase1Value}),
-					false,
-					200,
-				},
-			},
+			"empty secret",
+			newRequest(secretStore, utils.SimpleKeyValue{emptySecret, ""}),
+			newResponse(secretStore, utils.SimpleKeyValue{emptySecret, ""}),
+			false,
+			200,
+			"", // no error
 		},
 		{
-			// No comma since this will become the name of the test without spaces.
-			"Test unallowed secret",
-			[]testStep{
-				{
-					"get",
-					newRequest(utils.SimpleKeyValue{testCase2Key, ""}),
-					newResponse(utils.SimpleKeyValue{testCase2Key, ""}),
-					true,
-					403,
-				},
-			},
+			"allowed secret",
+			newRequest(secretStore, utils.SimpleKeyValue{allowedSecret, testCase1Value}),
+			newResponse(secretStore, utils.SimpleKeyValue{allowedSecret, testCase1Value}),
+			false,
+			200,
+			"", // no error
+		},
+		{
+			"unallowed secret",
+			newRequest(secretStore, utils.SimpleKeyValue{unallowedSecret, ""}),
+			newResponse("", utils.SimpleKeyValue{unallowedSecret, ""}),
+			true,
+			403,
+			"PERMISSION_DENIED",
```

Comment:
Oh that's interesting. Code suggests we expect "ERR_PERMISSION_DENIED" ? If I'm reading the correct line? https://github.com/dapr/dapr/blob/b6ea6f59231c50d2ee054b2a1f5b8d36c83db79c/pkg/http/api.go#L501

Reply:
Oh, I see we're using require.Contains().

Reply:
So, it is not essential that you change this, but it wouldn't hurt for consistency sake.